ui/webui/src/components/installation/InstallationProgress.scss

```diff
@@ -1,4 +1,8 @@
-.installation-progress-status {
+.pf-c-empty-state__content {
+    width: 100%
+}
+
+installation-progress-status {
```

Comment:
You are missing a dot here. I think that's why the big icon with the question mark is no longer red.

Reply:
Thanks, fixed! :)